collector/src/main/resources/logback-spring.xml

```diff
@@ -94,6 +94,7 @@
     <!-- Production environment configuration -->
     <springProfile name="prod">
         <root level="INFO">
+            <appender-ref ref="ConsoleAppender"/>
```

Comment:
remove this pls

Reply:
hi，I will fix this later, I am trying to reproduce and fix the e2e instability issue